packages/create-tina-app/src/util/install.ts

```diff
@@ -80,12 +80,10 @@ export function install(
        */
       args = ['install']
       if (!isOnline) {
-        console.log(chalk.yellow('You appear to be offline.'))
+        log.warn('You appear to be offline.')
         if (packageManager === 'yarn') {
-          console.log(chalk.yellow('Falling back to the local Yarn cache.'))
+          console.warn('Falling back to the local Yarn cache.')
```

Comment:
should this be `log.warn`?
```suggestion
          log.warn('Falling back to the local Yarn cache.')
```